### PR TITLE
Added Tim Sort [Python]

### PR DESCRIPTION
**Pull request by XavierChanth**
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

Fixes #{ISSUE_NUMBER}

**By submitting this pull request I confirm I've read and complied with the
below declarations.**

- [x] I have read the [Contribution guidelines](https://github.com/NITSkmOS/Algorithms/blob/master/CONTRIBUTING.md) and I am confident that my PR reflects them.
- [x] I have followed the [commit guidelines](https://github.com/NITSkmOS/Algorithms/blob/master/CONTRIBUTING.md#write-good-commit-messages) for this project.
- [x] My code follows the [standard code structure](https://github.com/NITSkmOS/Algorithms/blob/master/CONTRIBUTING.md#samples-code).
- [x] This pull request has a descriptive title. For example, `{Tag}: Add {Algorithm/DS name} [{Language}]`, not `Update README.md` or `Added new code`.
- [x] This pull request will be closed if I fail to update it even once in a continuous time span of 7 days.
- [x] This pull request shall only be reviewed and merged once all the checks passes. No maintainer or supporter shall be obliged to review it before this condition is met.
- [x] I have mentioned the issue number correctly (with hyperlink) in this pull request description.

After you submit your pull request, **DO NOT** click the 'Update Branch' button.

### CODE REVIEW

File: tim_sort/Python/tim_sort.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+
+def main():
+    arr = [12, 96, 24, 48, 36, 60, 84, 72]    # Declaring a sample array
+    tim_sort(arr)  # Calling the bubbleSort function and passing the array
+    print('Sorted array is: {}'.format(' '.join(map(str, arr))))
+
+
+if __name__ == '__main__':
+    main()
```
Comment (sangamcse) [inline tim_sort/Python/tim_sort.py]: Line contains following spacing inconsistencies:
- No newline at EOF.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpc083vmnv/tim_sort/Python/tim_sort.py
+++ b/tmp/tmpc083vmnv/tim_sort/Python/tim_sort.py
@@ -10,4 +10,4 @@
 
 
 if __name__ == '__main__':
-    main()+    main()
```

File: tim_sort/Python/tim_sort.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+
+def tim_sort(arr):  # Function that performs the tim sort
+   arr.sort()
```
Comment (sangamcse) [inline tim_sort/Python/tim_sort.py]: The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpc083vmnv/tim_sort/Python/tim_sort.py
+++ b/tmp/tmpc083vmnv/tim_sort/Python/tim_sort.py
@@ -1,6 +1,6 @@
 
 def tim_sort(arr):  # Function that performs the tim sort
-   arr.sort()
+    arr.sort()
 
 
 def main():
```

File: tim_sort/Python/tim_sort.py (hunk 1 above)
Comment (sangamcse) [inline tim_sort/Python/tim_sort.py]: The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpc083vmnv/tim_sort/Python/tim_sort.py
+++ b/tmp/tmpc083vmnv/tim_sort/Python/tim_sort.py
@@ -10,4 +10,4 @@
 
 
 if __name__ == '__main__':
-    main()+    main()
```

File: tim_sort/Python/tim_sort.py (hunk 2 above)
Comment (sangamcse) [inline tim_sort/Python/tim_sort.py]: E111 indentation is not a multiple of four

*Origin: PycodestyleBear (E111), Section: `all.autopep8`.*

File: tim_sort/Python/tim_sort.py (hunk 1 above)
Comment (sangamcse) [inline tim_sort/Python/tim_sort.py]: W292 no newline at end of file

*Origin: PycodestyleBear (W292), Section: `all.autopep8`.*

File: tim_sort/Python/tim_sort.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (XavierChanth) [reply tim_sort/Python/tim_sort.py]: It is but I have the real algo coming soon so people can see how they work

[the remark this answers is not in the document]
Comment (XavierChanth) [reply tim_sort/Python/tim_sort.py]: I just wanted to open the PR so that no one would beat me to adding tim_sort

**Pull request closed without merging**